modules/db.ts

```diff
@@ -11,9 +11,27 @@ export interface Item {
   createdAt?: string;
   updatedAt?: string;
 }
+
+export interface List {
+  id?: number;
+  name: string;
+  isActive?: boolean;
+  createdAt?: string;
+  updatedAt?: string;
+}
+
+// join table for relationship (many to many) between item and list
+export interface ItemList {
```

Comment:
It might be handy to assign priority to lists? Numbers might get confusing if we require them to be sequential and one list already holds number 1, for example - and it would be annoying for the user to have to go to that list, remove the priority, just so it's free for a new list. Probably just strings like "high", "medium", "low"